tools/db_crashtest.py

```diff
@@ -120,6 +120,9 @@
     "optimize_filters_for_memory": lambda: random.randint(0, 1),
     "partition_filters": lambda: random.randint(0, 1),
     "partition_pinning": lambda: random.randint(0, 3),
+    # `get_update_since_one_in` requires no sequence number hole in all WALs
+    # To simplify santization, we enable or disable `get_update_since_one_in` consistently across runs
```

Comment:
Hm that makes db_crashtest.py harder to use, compared to changing it per run.

If we save the `GetLatestSequenceNumber()` at startup and use that as a lower bound for `GetUpdatesSince()`'s `seq_number`, does that allow us to ignore what past runs did?